lib/task_listener.js

```diff
@@ -77,6 +77,11 @@ export default class TaskListener extends EventEmitter {
   }
 
   async listenForCancelEvents() {
+    // Do not listen for events if pulse configuration is not supplied
```

Comment:
Definitely log something with `[alert-operator]`. this shouldn't happen... And it's rather bad if it gets deployed like this.